src/lib/components/pagination.svelte

```diff
@@ -27,50 +26,62 @@
             ...(end < total - 1 ? ['...', total] : end < total ? [total] : [])
         ];
     }
+
+    function getLink(page: number): string {
+        const url = new URL($pageStore.url);
+        if (page === 1) {
+            url.searchParams.delete('page');
+        } else {
+            url.searchParams.set('page', page.toString());
+        }
+
+        return url.toString();
+    }
```

Comment:
If we're going to be using searchParams for multiple filters and what not, maybe we could offload this to a helper?

e.g. 
```
function getSearchParamLink(param: string, value: string): string {
    const url = new URL($pageStore.url);
    if (value === '') {
        url.searchParams.delete(param);
    } else {
        url.searchParams.set(param, value);
    }
    return url.toString();
}
```